apps/admin-gui/src/app/shared/components/sponsored-members-list/sponsored-members-list.component.html

```diff
@@ -60,6 +60,21 @@
               mat-icon-button>
               <mat-icon>more_horiz</mat-icon>
             </button>
+            <span
+              *ngIf="passwdResetAuth(sponsoredMember)"
+              [matTooltip]="'SHARED.COMPONENTS.SPONSORED_MEMBERS_LIST.PASSWORD_RESET' | translate"
+              [matTooltipPosition]="'above'"
+            >
+              <button
```

Comment:
Since you are not showing the dialog immediately and you are getting the logins, you should disable this button until the request is finished. This way, the user could click it multiple times because he would not be shown anything asap he clicks the button. Once the request has finished (or when the dialog is closed), you should enable this button.